src/main/java/dev/hephaestus/seedy/mixin/world/biome/source/MultiNoiseBiomeSourceMixin.java

```diff
@@ -42,6 +43,7 @@ private void replaceSeed(long seed, List<Pair<Biome.MixedNoisePoint, Supplier<Bi
 
 	@Redirect(method = "<init>(JLjava/util/List;Lnet/minecraft/world/biome/source/MultiNoiseBiomeSource$NoiseParameters;Lnet/minecraft/world/biome/source/MultiNoiseBiomeSource$NoiseParameters;Lnet/minecraft/world/biome/source/MultiNoiseBiomeSource$NoiseParameters;Lnet/minecraft/world/biome/source/MultiNoiseBiomeSource$NoiseParameters;Ljava/util/Optional;)V", at = @At(value = "NEW", target = "net/minecraft/world/gen/ChunkRandom"))
 	private ChunkRandom useOurSeed(long seed) {
-		return new ChunkRandom(this.seed);
+		if (counter == 4) counter = 0;
+		return new ChunkRandom(this.seed + counter++);
 	}
```

Comment:
Could you explain why you did this? It seems to me all this does is change it so that the `ChunkRandom` uses a different seed from the actual world seed, meaning that these biomes *are* generating, just not within a distance that locatebiome can detect in the seeds that have been tested.

Reply:
Moving my answer as it was not where I intended:
My goal here is to mimic the vanilla behavior: if you log the seed passed as an argument you can see there are 4 new ChunkRandom called with the values world_seed, world_seed+1, world_seed+2 and world_seed+3.